python/sparkdl/transformers/tf_tensor.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017 Databricks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numpy as np
+import tensorflow as tf
+import tensorframes as tfs
+
+from pyspark.ml import Transformer
+from pyspark.ml.param import Param, Params
+from pyspark.sql.functions import udf
+
+from sparkdl.transformers.param import (
+    keyword_only, HasInputCol, HasOutputCol, SparkDLTypeConverters)
+import sparkdl.graph.utils as tfx
+from sparkdl.graph.builder import IsolatedSession
+
+class TFOneDimTensorTransformer(Transformer, HasInputCol, HasOutputCol):
+    """
+    Applies the TensorFlow graph to the array column in DataFrame.
+
+    Restrictions of the current API:
+
+    We assume that 
+    - All graphs have a "minibatch" dimension (i.e. an unknown leading
+      dimension) in the tensor shapes.
+    - Input DataFrame has an array column where all elements have the same length
+
+    .. note:: The input tensorflow graph should have appropriate weights constantified,
+              since a new session is created inside this transformer.
+    """
+
+    graphFunction = Param(Params._dummy(), "graphFunction",
```

Comment:
We may not need all the params the other API has, but if that is the case, we should have this API start with params that are a strict subset of the other API.